draft-ietf-sipcore-sip-push.xml

```diff
@@ -358,7 +363,7 @@ align="center"><artwork>
   ; COLON as defined in RFC 3261
   
   The format and semantics of pns-param is specific to a given 
```

Comment:
`pns-param` -> `pn-param`

Reply:
Thought I fixed that... Strange... Anyway, I'll fix it :) Thanks!